.mergify.yml

```diff
@@ -0,0 +1,7 @@
+pull_request_rules:
+  - name: Automatic merge ⬇️ on approval ✔
+    conditions:
+      - "#approved-reviews-by>=1"
```

Comment:
This condition is not going to suffice, we want to make sure that unit tests pass before merging the command.
When a PR is created in Github, it does not have any checks. Then Github Actions or CentOS CI notices the PR and reports that CI is running. Only at that point will mergify notice that tests are not passing yet, but between the PR creation and the first CI status report there's a window where the PR seems to be correct (no failure). If somebody approves the PR during this window (which can be big if CI is slow that day), there's a risk of Mergify merging the PR before all tests have passed.
It is therefore recommended to list the required check that need to pass in the conditions.
See [Mergify's example](https://docs.mergify.io/examples/#automatic-merge-when-ci-works-and-approving-reviews) for this use case.

Reply:
Also, I'd like to merge PRs only if the review(s) was done by someone in the team (which will be `@fedora-infra/data-team` in GitHub)